docs/07-faq/022-supported-provisioning-engines.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Which provisioning engines are supported by Wing?
+id: supported-provisioning-engines
+keywords: [faq, supported provisioning engines, winglang, Wing programming language, Wing language, Terraform, AWS CDK, Pulumi]
+---
+
+Wing was built from the ground up to make it an ideal choice for building applications on any cloud, using a varaity of provisioning engines.
+Since wing is in early stages of developemnt, it officially only supports the Terraform provisioning engine at the moment. AWS CDK is supported by the community.
```

Comment:
```suggestion
Wing was built from the ground up to make it an ideal choice for building applications on any cloud, using a varaity of provisioning engines.
Since Wing is in early stages of development, it officially only supports the Terraform provisioning engine. AWS CDK (based on CloudFormation) is also supported by the community.
```